### DISCUSSION TITLE
suction base gripper？

### DISCUSSION THREAD

**Discussion by v-wewei** [bulletphysics/bullet3#3442] [General]
Is there any example for a suction base gripper?

**Comment by NiranthS** [comment]
You can try to imitation suction mechanism using applyExternalForce function of PyBullet

**Comment by v-wewei** [comment]
@NiranthS yet, how to imitation the seal suction mechanism in pybullet?

**Comment by NiranthS** [comment]
I am not sure what seal suction is.
But the suction type of mechanism can be shown by applying the equivalent amount of force at the gripper.
I think simulating actual fluid flow is not possible in PyBullet.
